tests/js/common/shell/shell-pregel-components.js

```diff
@@ -36,380 +36,382 @@ var internal = require("internal");
-          ["B20", "B22"],
-          ["C1", "C2"],
-          ["C2", "C3"],
-          ["C3", "C4"],
-          ["C4", "C5"],
-          ["C4", "C7"],
-          ["C5", "C6"],
-          ["C5", "C7"],
-          ["C7", "C8"],
-          ["C8", "C9"],
-          ["C8", "C10"],
-        ];
-
-        const vertices = new Set(edges.flat());
-
-        try {
-          graph_module._drop(problematicGraphName, true);
-        } catch (err) { }
-
-        const graph = graph_module._create(problematicGraphName, [graph_module._relation(e, v, v)]);
-
-        vertices.forEach(vertex => {
-          graph[v].save({ _key: vertex });
-        });
-
-        edges.forEach(([from, to]) => {
-          graph[e].save({ _from: `${v}/${from}`, _to: `${v}/${to}` });
-        });
-      }
-    },
-
-    ////////////////////////////////////////////////////////////////////////////////
-    /// @brief tear down
-    ////////////////////////////////////////////////////////////////////////////////
-
-    tearDownAll: function () {
-      graph_module._drop(graphName, true);
-      graph_module._drop(problematicGraphName, true);
-    },
-
-    testWCC: function () {
-      var pid = pregel.start("wcc", graphName, { resultField: "result", store: true });
-      var i = 10000;
-      do {
-        internal.sleep(0.2);
-        let stats = pregel.status(pid);
-        if (stats.state !== "loading" && stats.state !== "running" && stats.state !== "storing") {
-          assertEqual(stats.vertexCount, numComponents * n, stats);
-          assertEqual(stats.edgeCount, numComponents * (m + n), stats);
-
-          let c = db[vColl].all();
-          let mySet = new Set();
-          while (c.hasNext()) {
-            let doc = c.next();
-            assertTrue(doc.result !== undefined, doc);
-            mySet.add(doc.result);
-          }
-          assertEqual(mySet.size, numComponents);
-
-          break;
-        }
-      } while (i-- >= 0);
-      if (i === 0) {
-        assertTrue(false, "timeout in WCC execution");
-      }
-    },
-
-    testWCC2: function() {
-      const v = 'problematic_components';
-
-      if (internal.isCluster()) {
-        return;
-      }
-
-      // weakly connected components algorithm
-      var handle = pregel.start('wcc', problematicGraphName, {
-        maxGSS: 250, resultField: 'component'
-      });
-
-      while (true) {
-        var status = pregel.status(handle);
-        if (status.state !== 'loading' && status.state !== 'running' && status.state !== 'storing') {
-          console.log(status);
-          break;
-        } else {
-          console.log('Waiting for Pregel result...');
-          internal.sleep(1);
-        }
-      }
-
-      const counts = db._query(
-        `FOR vert IN @@v
+    return {
+
+        ////////////////////////////////////////////////////////////////////////////////
+        /// @brief set up
+        ////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
Pointless if it doesn't say what it sets up.

Reply:
Removed